CHANGELOG

```diff
@@ -1,3 +1,10 @@
+2016-08-30    0.16.2
+--------------------
+
+Enhancements:
+
+  * add support for cp35 and cp36 in _guess_platform_abi (#233)
```

Comment:
Nit: I think this change only adds support for cp36 (cp35 was already supported).